src/test/java/uk/gov/hmcts/divorce/notification/pin/PinGenerationServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.divorce.notification.pin;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+@ExtendWith(MockitoExtension.class)
+public class PinGenerationServiceTest {
+
+    @InjectMocks
+    private PinGenerationService pinGenerationService;
+
+    @Test
+    public void generatePinTest() {
+        String result = pinGenerationService.generatePin();
+
+        assertThat(result.length(), is(8));
```

Comment:
could you please use AssertJ as that would be consistent with what we are using in the project